success_stories/styles.css

```diff
@@ -0,0 +1,210 @@
+    font-size: 1rem;
+    font-weight: 300;
+    line-height: 1.8;
+    color: #000000;
+    margin-left: 5px;
+}
+
+
+/* -------------------- mentee and mentor details ------------------ */
+
+.details-box .details, .mentee .details {
+    font-size: 13px;
+    color: #000000;
+    margin-bottom: 0;
+    
+}
+
+.details-box .name, .mentee .name {
+    font-weight: bolder;
+}
+
+/* -----------------------------mentee -------------------------- */
+.mentee-details {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    text-align: center;
+}
+
+.mentee-details .details, .mentee-details .name {
+    margin-left: 0px;
+    margin-bottom: 0px;
+}
+
+.mentee-details .name {
+    margin-top: 5px;
+    font-size: 16px;
+}
+
+.mentee-details .details {
+    margin-top: 4px;
+}
+
+.mentee-profile-pic {
+    height: 100%;
+    margin-top: 27px;
+    margin-bottom: 2px;
+}
+
+.mentee-profile-pic .img {
+    border-radius: 50% !important;
+    width: 100px;
+    height: 100px;
+    border-radius: 50%;
+    border: 5px solid #EAECEE;
+}
+
+/* ----------------------------mentor ------------------------- */
+
+.mentor-details-title {
+    padding-top: 10px;
+    font-weight: bolder;
+    color: #000000;
+    font-size: 13px;
+    margin-bottom: 3px;
+}
+
+.mentor-details {
+    display: flex;
+    flex-wrap: inherit;
+    padding: 5px 15px 5px 15px;
+    margin-left: 4px;
+    margin-top: 4px;
+}
+
+.mentor-details .details,
+.mentor-details .name {
+    margin-left: 0px;
+    margin-bottom: 0px;
+}
+
+.mentor-details .name {
+    font-size: 14px;
+    margin-bottom: -1px;
+}
+
+.mentor-details .details{
+    line-height: 1.2rem;
+    margin-top: 4px;;
+}
+
+.mentor-profile-pic {
+    height: 100%;
+    margin-right: 10px;
+}
+
+.mentor-profile-pic .img {
+    border-radius: 50% !important;
+    width: 50px;
+    height: 50px;
+    margin-left: -15px;
+    border: 5px solid #EAECEE;
+    border-radius: 50%;
+}
+
+.mentor {
+    margin-left: 0px;
+}
+
+/* --------------------- skeleton loading ----------------------------- */
+.skeleton-storycard-common {
+    opacity: .7;
+    animation: skeleton-card-loading 1s linear infinite alternate;
+
+}
+
+@keyframes skeleton-card-loading {
+    0%{
+        background-color: hsl(210, 11%, 85%);
+    }
+
+    100%{
+        background-color: hsl(210, 11%, 93%);
+    }
+}
+
+.skeleton-storycard-text, .skeleton-storycard-text-short, 
+.skeleton-storycard-text-highlight, .skeleton-storycard-text-details {
+    margin-bottom: 0.5rem;
+    border-radius: 0.25rem;
+}
+
+.skeleton-storycard-text {
+    width: 100%;
+    height: 1rem;
+}
+
+.skeleton-storycard-text-short {
+    width: 20%;
+    height: 1rem;
+}
+
+.skeleton-storycard-text-highlight {
+    height: 1.5rem;
+}
+
+.skeleton-storycard-text-details {
+    width: 50px;
+    height: 0.8rem;
+    margin-top: 10px;
+}
+
+.skeleton-storycard-text:last-child {
+    margin-bottom: 0;
+    width: 80%;
+}
```

Comment:
Add a new line at the bottom of this file also @DilinaBandara

Reply:
Hi @YoshithaRathnayake  thank you for the comment! As there are a few more changes to be made to the initial structure we are hoping to complete it with @Madhawa97 and then add more changes. Thanks!

Reply:
Hi @DilinaBandara. This CSS file is in the `/success_stories `directory. Is it intentional to add these changes?

Reply:
we don't need this right? @DilinaBandara @Madhawa97

Reply:
yes @anjula-sack. If you use any of the styles from `/success_stories/styles.css` you can copy them into the CSS file for this page. @DilinaBandara

Reply:
Also, There is another `/success_stories/` directory added outside the `scholarx` directory. @DilinaBandara

Reply:
yes, Remove this CSS file